queries/deployments.sql

```diff
@@ -27,7 +27,12 @@ FROM(
       time_created,
       CASE WHEN source = "cloud_build" then JSON_EXTRACT_SCALAR(metadata, '$.substitutions.COMMIT_SHA')
            WHEN source like "github%" then JSON_EXTRACT_SCALAR(metadata, '$.deployment.sha')
-           WHEN source like "gitlab%" then JSON_EXTRACT_SCALAR(metadata, '$.commit.id') end as main_commit
+           WHEN source like "gitlab%" then JSON_EXTRACT_SCALAR(metadata, '$.commit.id') end as main_commit,
+      CASE WHEN source = "cloud_build" THEN []
+           WHEN source LIKE "github%" THEN ARRAY(
```

Comment:
Let's simplify this to be something like

```
CASE WHEN source LIKE "github%" THEN ARRAY( ...) 
ELSE [] 
END as additional_commits
```

We want to make sure that as people add integrations, it defaults to a value.